ftw/linkchecker/command/checking_links.py

```diff
@@ -0,0 +1,51 @@
+from AccessControl.SecurityManagement import newSecurityManager
+from Testing.makerequest import makerequest
+from zc.relation.interfaces import ICatalog
+from zope.component import getUtility
+from zope.component.hooks import setSite
+from zope.globalrequest import setRequest
+import AccessControl
+
+
+def get_plone_sites_information(app):
+    # make fake request to be able to traverse
+    app = makerequest(app)
+    setRequest(app.REQUEST)
+    overview_view = app.unrestrictedTraverse('plone-overview')
```

Comment:
Is this a stock-Plone view? Didn't know this one before, but it seems to work.
But an alternative would be to simply list the objects inside the `app` (which is a container / folderish object):

(untested)
```python
from Products.CMFPlone.interfaces import IPloneSiteRoot

# ...
plone_sites = [obj for obj in app.objectValues() if IPloneSiteRoot.providedBy(obj)]
```
(which doesn't involve the use of a browser view which might change between Plone versions)